cmd/reconciler/start/service/component_integration_test.go

```diff
@@ -0,0 +1,414 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	eventLog "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/eventing/log"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/kyma-incubator/reconciler/pkg/model"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes/progress"
+)
+
+type NoOpReconcileAction struct {
+	WaitTime time.Duration
+}
+
+// Run reconciler Action logic for Eventing. It executes the Action steps in order
+// and returns a non-nil error if any step was unsuccessful.
+func (a *NoOpReconcileAction) Run(context *service.ActionContext) (err error) {
+	// prepare logger
+	contextLogger := eventLog.ContextLogger(context, eventLog.WithAction("no-op-action"))
+	contextLogger.Infof("Waiting to simulate Op...")
+	time.Sleep(a.WaitTime)
+	return nil
+}
+
+func (s *reconcilerIntegrationTestSuite) setDefaultValuesFromTestCase(testCase *reconcilerIntegrationTestCase) {
+	if testCase.model.Namespace == "" && testCase.settings.namespace != "" {
```

Comment:
Why we do overwrite from settings only when original value is empty? Why not allow force overwrite?

Reply:
I just did this for more transparent behavior, I will add this.